app/models/api_token.rb

```diff
@@ -3,7 +3,7 @@ class ApiToken < ActiveRecord::Base
 
   before_create :generate_token
 
-  belongs_to :approval
+  belongs_to :approval, class_name: 'Approvals::Individual'
```

Comment:
Out of curiosity, why is this needed? Does it enforce what class can be assigned?